waiter/test/waiter/scheduler/marathon_test.clj

```diff
@@ -146,43 +147,43 @@
                           :marathon-response {
                                               :framework-id "F123445"
                                               :app {
-                                                    :id "test-app-1234",
```

Comment:
Why did all of these change?

Reply:
Since I am passing a single `service-id->service-description` map but want different protocols in the tests, I ended up changing their service-ids so I could configure them.

Reply:
I changed the tests to make `service-id->service-description` test case specific.